### ISSUE TITLE
Retrospectives

### ISSUE THREAD

**Issue by juliahayward** [red-gate/Tech-Radar#100]
As we have standups on our Adopt list, we really should add Retros as well...

**Comment by fffej** [comment]
No disagreements here - could you make a PR and I'll merge it @juliahayward ?

**Comment by nyctef** [comment]
+1 for adding retros, seems sensible

We might want to write something about the kinds of retrospectives we do:
* For a while on the masketeers team we experimented with using a private slack channel as a kind of suggestions box for retro topics (and then scheduling time as needed to deal with them). We found this didn't work too well for us, though - actually having an hour or two every couple of weeks as forced "stop and think" time ended up being more useful.
* I think most of the retros I've been to are either a [lean coffee](https://medium.com/agile-outside-the-box/lean-coffee-facilitator-s-guide-d79d9f13d0a9)-style "talk about what you want" or use some kind of gimmick (eg mad/sad/glad, winds&anchors, sprint review, draw a timeline, etc) to generate things to talk about. Do we prefer one style over the other? Do we just choose a different style for each retrospective to try and get as much coverage as possible?
